src/pykiso/test_result/assert_step_report.py

```diff
@@ -355,6 +355,20 @@ def _parse_timestamp(timestamp: float) -> str:
     return dt.strftime("%d/%m/%y %H:%M:%S")
 
 
+def is_test_success(test: List) -> bool:
+    """Check if test was successful.
+
+    :param tests: test
+    :return: True if each step in a test was successful else False
+    """
+    return not any([not step["succeed"] for step in test])
```

Comment:
```suggestion
    return all(step["succeed"] for step in test)
```
This seems equivalent and more readable